src/components/notebook-utils.js

```diff
@@ -58,6 +69,13 @@ export const notebookNameInput = ({ inputProps, ...props }) => h(ValidatedInput,
   }
 })
 
+export const tools = { rstudio: { label: 'RStudio', ext: 'rmd' }, jupyter: { label: 'Jupyter', ext: 'ipynb' } }
```

Comment:
Is the `ext` case insensitive? RStudio is generally `.Rmd`

Reply:
hmm... it is. I've just been using test files i made with `.rmd`, ill fix this